packages/gamut/src/Form/__tests__/SelectDropdown-test.tsx

```diff
@@ -108,4 +119,23 @@ describe('Select', () => {
     const menuList = wrapper.find('MenuList');
     expect(menuList.getDOMNode()).toHaveStyle('max-height : 8rem');
   });
+
+  it('renders a dropdown with icons', () => {
+    const { wrapper } = renderWrapper({
+      options: optionsIconsArray,
+      size: 'small',
+    });
+
+    wrapper.find('DropdownIndicator').first().simulate('mouseDown', {
+      button: 0,
+    });
+
+    const icon1 = wrapper.find(DataTransferVerticalIcon).text();
+    const icon2 = wrapper.find(CalendarIcon).text();
+    const icon3 = wrapper.find(EarthIcon).text();
+
+    expect(icon1).toBe('Data Transfer Vertical Icon');
+    expect(icon2).toBe('Calendar Icon');
+    expect(icon3).toBe('Earth Icon');
```

Comment:
If for whatever reason we stop rendering icon titles in the svg (we might since they are always hidden) these would break.

Since we're using enzyme here we can test this just by checking their presence :)
```suggestion
    expect(wrapper.find(DataTransferVerticalIcon)).toBeDefined();
    expect(wrapper.find(CalendarIcon)).toBeDefined();
    expect(wrapper.find(EarthIcon)).toBeDefined();
```